src/libs/ReportActionsUtils.ts

```diff
@@ -214,15 +214,53 @@ function getSortedReportActions(reportActions: ReportAction[] | null, shouldSort
         return (first.reportActionID < second.reportActionID ? -1 : 1) * invertedMultiplier;
     });
 
-    // If shouldMarkTheFirstItemAsNewest is true, label the first reportAction as isNewestReportAction
-    if (shouldMarkTheFirstItemAsNewest && sortedActions?.length > 0) {
-        sortedActions[0] = {
-            ...sortedActions[0],
-            isNewestReportAction: true,
-        };
+    return sortedActions;
+}
+
+/**
+ * Returns the range of report actions from the given array which include current id
+ * the range is consistent
```

Comment:
Did you mean "continuous" here?

I think the comment in its entirety should be something like this:

```
/**
 * Returns the largest gapless range of reportActions including a the provided reportActionID,
 * where a "gap" is defined as a reportAction's `previousReportActionID` not matching the previous reportAction in the sortedReportActions array.
 *
 * See unit tests for example of inputs and expected outputs.
 */
```

Reply:
And then as alluded to in that comment, let's add unit tests to cover this function – it will double as documentation for how it should work. I wrote up some examples in that comment before realizing it would be just as simple and more valuable to write those examples as unit tests:

```js
// Given these sortedReportActions
[
    {reportActionID: 1, previousReportActionID: null},
    {reportActionID: 2, previousReportActionID: 1},
    {reportActionID: 3, previousReportActionID: 2},

    // Note: there's a "gap" here because the previousReportActionID (4) does not match the ID of the previous reportAction in the array (3)
    {reportActionID: 5, previousReportActionID: 4},
    {reportActionID: 6, previousReportActionID: 5},

    // Note: another gap
    {reportActionID: 8, previousReportActionID: 7},
]

// Then given an input ID of 1, 2, or 3, the return value of this function would be:

[
    {reportActionID: 1, previousReportActionID: null},
    {reportActionID: 2, previousReportActionID: 1},
    {reportActionID: 3, previousReportActionID: 2},
]

// Given an input ID of 5 or 6, it would be:

[
    {reportActionID: 5, previousReportActionID: 4},
    {reportActionID: 6, previousReportActionID: 5},
]

// Given an input ID of 8, it would be:

[
    {reportActionID: 8, previousReportActionID: 7},
]

// Given an input of 4, 7, or any other number, it would return [].
```